locale/en.json

```diff
@@ -473,35 +473,94 @@
 		"checkout-block.Zambia": "Zambia",
 		"checkout-block.Zimbabwe": "Zimbabwe",
 		"checkout-block.back-to-offer-page": "Back to subscription selection",
+		"checkout-block.view-subscription-offers": "View subscription offers",
```

Comment:
@LauraPinilla , Just making a note here since this is the 'main' PR for the checkout block - We need these phrases in every language file even if they are not translated yet. If a customer tries to setup this new block and they use a locale other than `en` they will get errors when PB Engine tries to fetch the phrases.